boards/yarm/include/periph_conf.h

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright (C) 2020 Antonio Galea
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_yarm
+ * @{
+ *
+ * @file
+ * @brief       Peripheral MCU configuration for the Acmesystems
+ *              YARM board.
+ *
+ * @author      Antonio Galea <antonio.galea@gmail.com>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   GCLK reference speed
+ */
+#define CLOCK_CORECLOCK     (48000000U)
+
+/**
+ * @brief Enable the internal DC/DC converter
+ *        The board is equipped with the necessary inductor.
+ */
+#define USE_VREG_BUCK       (1)
+
+/**
+ * @name    Timer peripheral configuration
+ * @{
+ */
+static const tc32_conf_t timer_config[] = {
+    {   /* Timer 0 - System Clock */
+        .dev            = TC0,
+        .irq            = TC0_IRQn,
+        .mclk           = &MCLK->APBCMASK.reg,
+        .mclk_mask      = MCLK_APBCMASK_TC0 | MCLK_APBCMASK_TC1,
+        .gclk_id        = TC0_GCLK_ID,
+        .gclk_src       = SAM0_GCLK_8MHZ,
+        .flags          = TC_CTRLA_MODE_COUNT32,
+    }
+};
+
+/* Timer 0 configuration */
+#define TIMER_0_CHANNELS    2
+#define TIMER_0_ISR         isr_tc0
+#define TIMER_NUMOF         ARRAY_SIZE(timer_config)
+/** @} */
+
+/**
+ * @name    UART configuration
+ * @{
+ */
+static const uart_conf_t uart_config[] = {
+    {    /* Virtual COM Port */
```

Comment:
`stdio_cdc_acm` creates a virtual COM port on the USB interface, this is just a UART on some GPIO pins - or is there a USB - TTL connected to those?

Reply:
It's a plain UART, see the [pinout](https://www.acmesystems.it/pinout_yarm).

Reply:
So the comment is wrong.
But that pinout is nice, can you add it in the board's `doc.txt`

Reply:
Corrected the comment. Still need to add a doc.txt file.

Reply:
Added a doc.txt for the board.